websites/G/G2G/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "283757164923715587",
+		"name": "Mattis.#0001"
```

Comment:
Discriminator added by outdated pmd
```suggestion
		"name": "Mattis."
```

Reply:
Used yarn format on the terminal before, so i can't make it prettier.
And the discriminator is auto when you put your discord ID on the presence creating process

Reply:
The discriminator is a bug from an old version of the presence creating program.

Reply:
Unresolved